jsHelper/spicetifyWrapper.js

```diff
@@ -348,7 +349,7 @@ window.Spicetify = {
 })();
 
 (function addProxyCosmos() {
-	if (!Spicetify.Player.origin?._cosmos && !Spicetify.Platform?.Registry) {
+	if (!Spicetify.Player.origin || !Spicetify.Platform?.Registry) {
```

Comment:
That was intentional tho

Reply:
I get the idea was to wait for either `_cosmos` or `Registry`, but in this case `origin` may not be available before `Registry` which causes a TypeError when assigning `_cosmos` to `origin`
https://github.com/spicetify/cli/blob/eaa16bb2aed8f30bcd481aeac9af4ec66585a0eb/jsHelper/spicetifyWrapper.js#L440

Reply:
When did you get this issue because I have never seen TypeError in this instance